### DISCUSSION TITLE
Bed levelling with screws

### DISCUSSION THREAD

**Discussion by simonvez** [KlipperScreen/KlipperScreen#94] [General]
trying to understand the py script to manual bed levelling. I have no probe, all manual. I have a section defined as [bed_screws], but the klipperscreen will not go to where the screws are defined. It will offset them by 78 or somtething like that. when I use the klipper BED_SCREWS_ADJUST gcode command, it goes to where I define the [bed_screws]. It would be nice if the LCD would do the same.

**Comment by jordanruthe** [comment]
Can you post a KlipperScreen.log file after you go into the manual bed level panel and go to s location or two?

**Reply by apolkosnik** [reply]
@simonvez, I'm just guessing that you had only 3 screws defined under [bed_screws], in which case the coordinates are being computed and come out with pretty wide margins.

**Comment by alfrix** [comment]
the current version does respect the config much better, i consider this resolved

**Reply by solograppler** [reply]
Can you elaborate as I still have the same issue.

**Reply by alfrix** [reply]
https://klipperscreen.readthedocs.io/en/latest/Panels/Screws

**Reply by alfrix** [reply]
and to those who have both sections bed_screws and screws_tilt_adjust:
the probe should be accurate and give you the correct value, the idea of the button is double checking comparing with the test done on zcalibrate, why would you measure a point to adjust with the probe and check a different one manually, it doesn't make any sense
